README.md

```diff
@@ -27,32 +28,28 @@ GPflow 2.0 uses [TensorFlow 2.0](http://www.tensorflow.org) for running computat
 
 ### Install GPflow 2
 
-- From source
+#### Latest release from PyPI
 
-  With the release of _TensorFlow_ 2.1 and _Tensorflow Probability_ 0.9, you should
-  only need to run
+```bash
+pip install gpflow
+```
```

Comment:
anticipating the release already, just to confirm?

Reply:
Indeed. Is that okay?